root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -2850,6 +2850,42 @@ limited_link_type_combinations: [
     expected_relationship_type: ['downloadfree', 'streamingfree'],
             expected_clean_url: 'http://www.jamendo.com/album/56372',
   },
+  // Jaxsta
+  {
+                     input_url: 'http://www.jaxsta.com/profile/8a9bd5c6-1ce2-4f1d-bfdc-1a2383129c79/catalogue?role_group=Featured%20Artist&view=tile',
+             input_entity_type: 'artist',
+    expected_relationship_type: 'otherdatabases',
+            expected_clean_url: 'https://jaxsta.com/profile/8a9bd5c6-1ce2-4f1d-bfdc-1a2383129c79',
+       only_valid_entity_types: ['artist', 'label'],
+  },
+  {
+                     input_url: 'https://jaxsta.com/profile/5a50b1be-7473-4e51-ae4b-bb8d9ac13407/catalogue?view=tile',
+             input_entity_type: 'label',
+    expected_relationship_type: 'otherdatabases',
+            expected_clean_url: 'https://jaxsta.com/profile/5a50b1be-7473-4e51-ae4b-bb8d9ac13407',
+       only_valid_entity_types: ['artist', 'label'],
+  },
+  {
+                     input_url: 'https://jaxsta.com/recording/5b01140a-a82f-5f5c-aa81-93d61de00f94/releases',
+             input_entity_type: 'recording',
+    expected_relationship_type: 'otherdatabases',
+            expected_clean_url: 'https://jaxsta.com/recording/5b01140a-a82f-5f5c-aa81-93d61de00f94',
+       only_valid_entity_types: ['recording'],
+  },
+  {
+                     input_url: 'https://jaxsta.com/release/66512305-5fb7-5d43-a194-d5ed56c16cab/7c5528fe-96f7-5c0c-9af2-9b756b909436/variants',
+             input_entity_type: 'release',
+    expected_relationship_type: 'otherdatabases',
+            expected_clean_url: 'https://jaxsta.com/release/66512305-5fb7-5d43-a194-d5ed56c16cab/7c5528fe-96f7-5c0c-9af2-9b756b909436',
+       only_valid_entity_types: ['release'],
+  },
```

Comment:
This is linking to a “release variant” which matches our release.
Their “release” actually matches our release group.
This should be explained in ticket/pull request description/commit message.

Also it seems that it would be helpful to allow linking `https://jaxsta.com/release/66512305-5fb7-5d43-a194-d5ed56c16cab` for example, maybe formatted as `https://jaxsta.com/release/66512305-5fb7-5d43-a194-d5ed56c16cab/group/variants`, to a release group in case the current releases in the release group don’t match any of the current variants in Jaxsta.

Reply:
Hmm. I didn't notice there was a way to link to a specific RG-level page, because just doing https://jaxsta.com/release/66512305-5fb7-5d43-a194-d5ed56c16cab automatically loads a variant (hence adding it as release only). But we also don't want to clean up the variant, so I guess we should only convert to https://jaxsta.com/release/66512305-5fb7-5d43-a194-d5ed56c16cab/group/variants if it's given without a second UUID, right? (ideally we'd just clean up differently depending on whether the user is at a RG page or a release one, but I guess we don't have any way to do that...)

Reply:
There is no way to do it from their website, I edited the URL by hand. I thought it would be reasonable to replace the second UUID with `group` and add the subpath `/variants` but I now realize that the function `cleanup` only depends on the URL and not on the entity type being edited. So let’s give up this idea for now but still further explain the release mapping in commit message.